pkg/cmd/admin/registry/registry.go

```diff
@@ -300,21 +300,16 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
 		if err := restclient.LoadTLSFiles(config); err != nil {
 			return fmt.Errorf("registry does not exist; the provided credentials %q could not load certificate info: %v", opts.Config.Credentials, err)
 		}
-		insecure := "false"
-		if config.Insecure {
-			insecure = "true"
-		} else {
-			if len(config.KeyData) == 0 || len(config.CertData) == 0 {
-				return fmt.Errorf("registry does not exist; the provided credentials %q are missing the client certificate and/or key", opts.Config.Credentials)
-			}
+		if !config.Insecure && (len(config.KeyData) == 0 || len(config.CertData) == 0) {
+			return fmt.Errorf("registry does not exist; the provided credentials %q are missing the client certificate and/or key", opts.Config.Credentials)
 		}
 
 		secretEnv = app.Environment{
 			"OPENSHIFT_MASTER":    config.Host,
 			"OPENSHIFT_CA_DATA":   string(config.CAData),
 			"OPENSHIFT_KEY_DATA":  string(config.KeyData),
 			"OPENSHIFT_CERT_DATA": string(config.CertData),
-			"OPENSHIFT_INSECURE":  insecure,
+			"OPENSHIFT_INSECURE":  fmt.Sprintf("%v", config.Insecure),
```

Comment:
It would probably be better to use `%t` here because we want Insecure to be flagged as a bool.